### ISSUE TITLE
Whenever I say hello it does a goodbye statement

### ISSUE THREAD

**Issue by dell5463** [TheBlueChemical/neuralninechatbot#1]
So whenever I say "Hello" it would say my goodbye statements.

**Comment by Waddlo** [comment]
> So whenever I say "Hello" it would say my goodbye statements.

Yeah it's not perfect, it's designed to be a simple AI, so you might need to add more keywords in the intents.json. Here's the video for a better explanation. https://www.youtube.com/watch?v=1lwddP0KUEg
